### DISCUSSION TITLE
Prime Elite - Text Based BBC Elite Game

### DISCUSSION THREAD

**Discussion by SirNibble** [prirun/p50em#34] [Show and tell]
I've had some time on my hands ... Have fun!

https://github.com/SirNibble/Prime-Text-Elite

A port of txtelite.c from Ian Bell's site
http://elitehomepage.org/text/index.htm

Ported to Pascal on Rev 23 Primos. It might work on other revisions, but untested.

This started purely as an excerise to understand the infamous 'Goat Soup' Algorithm used to generate the planet descriptions. My 'C' is terrible, source initially started playing in Lazerus Pascal, but a sudden flash of inspiration/stupidty made me think of more era appropiate solutions and what fun it would have been to have even just this 'text adventure' version on Primos back-in-the-day.

So here we are!

It's not perfect, but has been an awful lot of fun to do, despite some of the 'restrictions' of Prime's Pascal.

Now working on Version 2 to be a bit prettier using the good ol' VT100 escape codes.

Developed using Notepad++ and uploaded via Kermit.

With help from 'Fully documented source code for Elite on the BBC Micro'
https://www.bbcelite.com/deep_dives/galaxy_and_system_seeds.html

Elite: "The game that couldn't be written"
https://youtu.be/lC4YLMLar5I

**Comment by drboone** [comment]
Neat!  I'll try it out.  Er, but I think you need to make the repo public. :)

**Comment by drboone** [comment]
Reading wikipedia, it seems like some aspects of this game bear similarities to the old BBS door games in the Tradewars family.

**Comment by SirNibble** [comment]
Ooops! Try it now! ;)

**Comment by drboone** [comment]
Got a chuckle out of the klm serial number banner. :)
